src/App.tsx

```diff
@@ -14,17 +15,19 @@ import RegisterPhoto from './domain/Photo/RegisterPhoto';
 function App() {
   return (
     <Routes>
-      <Route path={'/'} element={<Login />} />
-      <Route path={'/login'} element={<Login />} />
-      <Route path={'/mypage'} element={<MyPage />} />
-      <Route path={'/album'} element={<AlbumRouters />}>
-        <Route index element={<AlbumList />} />
-        <Route path={'edit'} element={<EditAlbum />} />
-        <Route path={'register'} element={<RegisterAlbum />} />
-      </Route>
-      <Route path={'/photo'} element={<PhotoRouters />}>
-        <Route index element={<PhotoList />} />
-        <Route path={'register'} element={<RegisterPhoto />} />
+      <Route element={<DefaultMobileLayout />}>
+        <Route path={'/'} element={<Login />} />
+        <Route path={'/Login'} element={<Login />} />
+        <Route path={'/MyPage'} element={<MyPage />} />
```

Comment:
```suggestion
        <Route path={'/login'} element={<Login />} />
        <Route path={'/my-page'} element={<MyPage />} />
```
url 네이밍에는 소문자와 하이픈('-')을 사용하는 kebab-case 규칙을 따르는게 권장된다고 합니다.
- Google의 URL 구조 가이드라인 : https://developers.google.com/search/docs/crawling-indexing/url-structure?hl=ko

Reply:
그렇구나.. 그래서 대부분의 웹사이트가 kebab-case를 따랐던 거군요

Reply:
이건 상준님이 도훈님이 말씀해주신거 리팩토링 하시면서 같이 해주실 수 있나요??
제가 작업했던 부분이 아니였는데 merge 하면서 따라 들어왔네요 ㅠ

Reply:
아 그런가요? 넵넵

Reply:
my-page할까 mypage할까 하다가 그냥 mypage했어요 ㅋㅋ